### PR TITLE
Extend WindowTests example to include switchable glazing with illuminance control, and other minor test changes

### PR DESCRIPTION
**Pull request by mjwitte**
Pull request overview
---------------------
1. Address #6913 by adding another zone to the WindowTests example file which has shading type SwitchableGlazing with shading control type MeetDaylightIlluminanceSetpoint. Added daylighting controls to existing zone DPW (double pane no shading) and new zone DPSG (double pane switchable glazing) plus additional output variables to confirm operation.

2. Moved all of the inactive performance test inputs to an "Inactive" subfolder.

3. Added to annual simulations to the integration/regression tests.

### Work Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [x] At least one of the following appropriate labels must be added to this PR to be consumed into the changelog:
   - Defect: This pull request repairs a github defect issue.  The github issue should be referenced in the PR description
   - Refactoring: This pull request includes code changes that don't change the functionality of the program, just perform refactoring
   - NewFeature: This pull request includes code to add a new feature to EnergyPlus
   - Performance: This pull request includes code changes that are directed at improving the runtime performance of EnergyPlus
   - DoNoPublish: This pull request includes changes that shouldn't be included in the changelog

### Review Checklist
This will not be exhaustively relevant to every PR.
 - [ ] Code style (parentheses padding, variable names)
 - [ ] Functional code review (it has to work!)
 - [ ] If defect, results of running current develop vs this branch should exhibit the fix
 - [ ] CI status: all green or justified
 - [ ] Performance: CI Linux results include performance check -- verify this
 - [ ] Unit Test(s)
 - C++ checks:
   - [ ] Argument types
   - [ ] If any virtual classes, ensure virtual destructor included, other things
 - IDD changes:
   - [ ] Verify naming conventions and styles, memos and notes and defaults
   - [ ] Open windows IDF Editor with modified IDD to check for errors
   - [ ] If transition, add rules to spreadsheet
   - [ ] If transition, add transition source
   - [ ] If transition, update idfs
 - [ ] If new idf included, locally check the err file and other outputs
 - [ ] Documentation changes in place
 - [ ] Changed docs build successfully
 - [ ] ExpandObjects changes?
 - [ ] If output changes, including tabular output structure, add to output rules file for interfaces

### CODE REVIEW

**Comment by mjwitte** [comment]
@Myoldmopar This is ready for review.

**Comment by mjwitte** [comment]
@Myoldmopar Would you be open to adding one or two annual simulations to the testfiles? e.g. 5ZoneAirCooled. And why are some of the 5ZoneAirCooled files being [run with CO weather](https://github.com/NREL/EnergyPlus/blob/d79a0f2f2cd90a2d57a44b3b3ceefbc05d87a0f3/testfiles/CMakeLists.txt#L29-L31) instead of Chicago?

**Review by Myoldmopar — APPROVED** [review]
Changes look reasonable.  Built locally and ran the modified test file.  In design-day-only, the only warning was the tabular outputs for a non-annual run.  In annual run, the only warning was that I used Chicago weather instead of Denver.  This is good to go.  Merging.

**Comment by Myoldmopar** [comment]
Let me rephrase myself a bit.  This is all good to go, but feel free to add a couple annual run cases to CI here before I merge.

**Comment by Myoldmopar** [comment]
I'd suggest renaming those files.  The annual version is probably going to overwrite the first one.  I'd just copy the file with _annual appended to the name -- even though there's no actual difference in the idf.  Then change the CMakeLists accordingly and there won't be any concern about overwritten results.

**Comment by mjwitte** [comment]
Should I do those with _ prefix so then they don't get duplicated in the install?

**Comment by Myoldmopar** [comment]
Excellent idea!
